### ISSUE TITLE
Save file and transfer highlights

### ISSUE THREAD

**Issue by sleyden-nuance** [ryu1kn/vscode-text-marker#58]
Hello,

I have a .json file in VSCode that I have highlighted appropriately and saved.  When I give this file to my colleague and he installs the plugin - he can not see any of the highlights I have added - why is this?  Is there anyway to save highlights to a file to be able to have your colleague see those highlights?  Please let me know.

Thanks,

Scott

**Comment by ryu1kn** [comment]
Highlights are saved as `textmarker.savedHighlights` either in your user settings or workspace settings depending on your choice. Have you given that bit of the setting to your colleague and still he cannot see the highlight restored?

**Comment by sleyden-nuance** [comment]
Yes I was able to save the workspace settings for "textmarker.savedHighlights" and provide those settings to my colleagues and they were able to see the highlights.  So each time I want to highlight selected code with a specific color and share with a colleague to perhaps change the highlight color or add more selected text highlights we must just share our workspace settings?  Is there anyway to just provide highlights with file rather than have to give both the file and "settings.json file?  Just seems like a lot of work to accomplish what I thought was a simple task.
